lib/jobs/ucs-discovery.js

```diff
@@ -210,6 +210,7 @@ function UcsDiscoveryJobFactory(
             id = [config.ucs, data.path];
         }
 
+        config.dn = data.path;
```

Comment:
Please change a better variable name, `dn` is too short and never tell anything.